### PR TITLE
request to rails for getting  work datetime

### PR DESCRIPTION
**Pull request by tz3**
replace function on middleware
and work datetimes getting from the rails

### CODE REVIEW

File: controllers/api/v1/public_controller.es6 [hunk 1]
```diff
@@ -40,28 +31,30 @@ let eventDecorator = (req, shop, data) => {
...
 router.get('/auth', (req, res) => {
-    auth.identification(req.param('token'))
-        .then(shop => {
-            res.status(200).json(shopDecorator(shop));
-        })
-        .catch(error => {
-            return res.status(401).json({error: error});
+    let shop = req.shop;
+    time_service()
+        .then(datetimes => {
```
Comment (in-78) [inline controllers/api/v1/public_controller.es6]: datetimes не стоит называть респонс

File: controllers/api/v1/public_controller.es6 [hunk 2]
```diff
@@ -40,28 +31,30 @@ let eventDecorator = (req, shop, data) => {
...
-            let logStream = fs.createWriteStream('./logs/event.log', {'flags': 'a+'});
-            logStream.end(JSON.stringify(logMessage) + "\n");
-            return res.status(201).json({status: 'ok'});
-        })
-        .catch(error => {
-            return res.status(401).json({error: error});
-        });
+    let shop = req.shop;
+    let logMessage = eventDecorator(req, shop, data);
+    let logStream = fs.createWriteStream('./logs/event.log', {'flags': 'a'});
```
Comment (in-78) [inline controllers/api/v1/public_controller.es6]: a+

File: controllers/api/v1/public_controller.es6 [hunk 3]
```diff
@@ -1,33 +1,24 @@
...
+dotenv.config();
 let router = Router();
 
-
-let time_service = (shop) => {
-    return {
-        from: new Date(),
-        to: new Date()
-    }
+let time_service = () => {
```
Comment (AndreyNenashev) [inline controllers/api/v1/public_controller.es6]: timeService

File: controllers/api/v1/public_controller.es6 [hunk 4]
```diff
@@ -1,33 +1,24 @@
...
+    let fullUrl = `http://${process.env.GETPARCEL_ADMIN_HOST}:${process.env.GETPARCEL_ADMIN_PORT}/api/v2/system_status`;
+    return request({
+        url: fullUrl,
+        json: true
+    })
 };
 
 let show_widget = () => {
```
Comment (AndreyNenashev) [inline controllers/api/v1/public_controller.es6]: showWidget

**Pull request merged**